### ISSUE TITLE
Integrate AI for hyper

### ISSUE THREAD

**Issue by gsdt** [zeit/hyper#8056]
**Is your feature request related to a problem? Please describe.**
Remembering complex terminal command syntax can be challenging. I would like to describe commands in natural language and have AI generate the appropriate command. I can then choose to run it or ask the AI to modify it.

**Describe the solution you'd like**
ChatGPT is widely used today, so it would be great to integrate it into our terminal for an enhanced experience.
For example, pressing a specific key combination could trigger a popup, allowing us to input requirements for the AI to process.

**Comment by sagarpanchal** [comment]
You can use warp or a separate hyper plugin, command or shell plugin that takes your query and returns command.
No need to bloat the terminal with unnecessary features as there are many shell plugins for auto suggestion and completion from history that are enough for most of the users.

**Comment by QR-Madness** [comment]
@sagarpanchal I agree. Hyper was created to be fast, integrated ML models are against the overall vision. @gsdt You should investigate using local LLM mini-models and filesystem agents for an extension if you'd like a full terminal AI suite.
